app/src/main/java/com/ncnf/database/builders/EventBuilder.java

```diff
@@ -88,8 +88,6 @@ public Event toObject(String uuid, @NotNull Map<String, Object> data){
         double lng = event.getLocation().getLongitude();
         String hash = GeoFireUtils.getGeoHashForLocation(new GeoLocation(lat, lng));
         map.put(GEOHASH_KEY, hash);
-        map.put(LAT_KEY, lat);
-        map.put(LNG_KEY, lng);
```

Comment:
I guess we do not store lat/long anymore, but a GeoHash ?

Reply:
Yeah I stored them at first as it was in the geoQuery tutorial, but it was actually used not for the geoHash but for an extra check that it is indeed in the radius. Since we already have a method that does it in MapUtilities, I removed those two fields and kept only the geHash